saged/predict_tissue.py

```diff
@@ -28,6 +30,70 @@
                      'Cerebellum', 'Cerebral_cortex', 'Salivary_Gland', 'Duodenum'
                      ]
 
+
+def objective(trial, train_list, supervised_config,
+              label_encoder, weighted_loss=False, device='cpu'):
+    losses = []
+
+    with open(supervised_config) as supervised_file:
+        supervised_config = yaml.safe_load(supervised_file)
+    supervised_model_type = supervised_config.pop('name')
+
+    # TODO there should be a better way to do this which allows future skl models
```

Comment:
But some sklearn models do have a learning rate parameter, right? I think hard-coding this is fine, unless there's a way to check if the model has a learning rate parameter directly (I can't personally think of a good way, so I'd probably just do what you're doing).

Reply:
> But some sklearn models do have a learning rate parameter, right?

True

> I think hard-coding this is fine, unless there's a way to check if the model has a learning rate parameter directly

Yeah, it's a hard fix to implement. I'll fix it if it becomes a problem but not until then